src/components/Book.js

```diff
@@ -1,30 +1,30 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/Books';
 import InputBook from './Form';
 
-function Book(props) {
-  const { title, author } = props;
+function Book() {
+  const dispatch = useDispatch();
+  const listBooks = useSelector((store) => store.books);
+  console.log(listBooks);
   return (
     <div>
       <ul>
-        <li>
-          {title}
-          {author}
-        </li>
+        {
+          listBooks.map((item) => (
+            <li key={item.id}>
+              {item.title}
+              <br />
+              {item.author}
+              <br />
+              <button className="removeBtn" type="button" onClick={() => dispatch(removeBook())} key={item.id}>Remove</button>
+            </li>
```

Comment:
- [ ] There should be another component, that renders a single book. Kindly create a new reusable component that represents a book, pass in `title` and `author` as props and render them in that component.

  Example
  ```
   <SingleBook title={item.title} author={item.author}>
  ```
  Then in the SingleBook component you can render this code accordingly.